public/table-sort.js

```diff
@@ -328,7 +328,9 @@ function tableSortJs(testingTableSortJS = false, domDocumentWindow = document) {
 
     th.addEventListener("click", function () {
       const columnData = [];
-      const tableRows = tableBody.querySelectorAll("tr");
+      const tableRows = Array.prototype.filter.call(tableBody.querySelectorAll("tr"), e => {
+        return e.style.display != 'none';
```

Comment:
nit: Could you change the inequality operator `!=` to use the strict inequality operator `!==`. 
nit: Would you be able to call `e` something more descriptive of what it is?. For example: `tableRow`.